test/utils/SafeCast.test.js

```diff
@@ -82,4 +82,35 @@ describe('SafeCast', async () => {
       );
     });
   });
+
+  describe('toInt256', () => {
+    const maxUint256 = new BN('0').notn(256);
+    const maxInt256 = new BN('1').shln(255).notn(256);
```

Comment:
These seem to be correct, but I find them a bit cryptic. What do you think about writing them in terms of powers instead of shifts? So maxInt256 would be `2^256 - 1`.

Reply:
Sure thing. I used shifts as it was cleaner (and clearer) to me, but I see that it can be a bit confusing specially considering all the rest are defined using powers of 2.